lib/menus/commands.js

```diff
@@ -391,7 +391,7 @@ const init = () => {
         $treeSelected.each((key, elem) => {
           const path = elem.getPath ? elem.getPath() : '';
           const localPath = path.replace(client.root.local, '');
-          const remotePath = Path.posix.normalize(atom.project.remoteftp.root.remote + localPath);
+          const remotePath = Path.posix.normalize(atom.project.remoteftp.root.remote + localPath.replace(/\\/g, '/'));
```

Comment:
I'd rather suggest this
```javascript
const remotePath = Path.posix.normalize((atom.project.remoteftp.root.remote + localPath).replace(/\\/g, '/'));
```

Reply:
I agree, that probably can't hurt. On my machine atom.project.remoteftp.root.remote was using / already, but it's not a bad idea to replace any \ with / there as well

Reply:
Okay, replace and I accept PR.

Reply:
all set, file is updated